src/js/components/PageHeaderTabs.js

```diff
@@ -38,15 +50,66 @@ class PageHeaderTabs extends React.Component {
     });
 
     return (
+      <ul className="page-header-navigation-tabs menu-tabbed">
+        {tabElements}
+      </ul>
+    );
+  }
+
+  getNavigationDropdownItems() {
+    const {tabs} = this.props;
+
+    if (tabs == null) {
+      return [];
+    }
+
+    return tabs.map((tab, index) => {
+      const {callback, isActive, label, routePath} = tab;
+      // We add 1 to index for the ID to avoid an ID of 0, due to coercion in
+      // the Dropdown component.
+      const id = index + 1;
+
+      return {
+        label,
+        id,
+        isActive: isActive || (routePath != null
+          && this.isRouteActive(routePath)),
+        callback,
+        routePath
+      };
+    });
+  }
+
+  handleNavigationItemSelection({callback, routePath}) {
```

Comment:
Done! I've added a note to the teamsync doc to make sure we reiterate this rule, I actually thought we weren't doing this anymore.